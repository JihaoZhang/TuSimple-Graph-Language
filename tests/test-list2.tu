int main()
{	
	int i;
 	list @{int} l;
 	list @{int} l1;
 	new l1;
 	l = @{11,22,33};
 	l += @{44,55,66};
 	print (l.length());
 	print (l[5]);
 	for (i = 0; i < l.length(); i+=1) {
 		print(l.get(i));
 	}
 	l.pop();
 	l.pop();
 	print(l.length());


 	for (i = 0; i < l.length(); i+=1) {
 		print(l.get(i));
 	}
 
 	return 0;
 } 