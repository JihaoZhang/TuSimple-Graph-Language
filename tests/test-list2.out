6
66
11
22
33
44
55
66
4
11
22
33
44